riscv/rocc.cc

```diff
@@ -7,6 +7,7 @@
 #define customX(n) \
   static reg_t c##n(processor_t* p, insn_t insn, reg_t pc) \
   { \
+    require_accelerator; \
```

Comment:
The ROCC extension can be in `EXT_STATE_OFF` even if other extensions are enabled (and hence XS != `EXT_STATE_OFF`). Shouldn't this only be checking for the state of ROCC and not all the other extensions too?

Reply:
Yes, you are right.